notebooks/IDC_segmentation_primer.ipynb

```diff
@@ -1,1927 +1 @@
-{
```

Comment:
<p>For itkWidgets, perhaps also show how manual and AI segmentations can be combined (via simple addition of the labelmaps) to create a tri-color labelmap that shows agreement, false-negative, and false-positive labels made by the AI?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
Added an example